types/coinbase/index.d.ts

```diff
@@ -600,7 +600,7 @@ export class Account implements Resource {
      * Lists account’s transactions.
      * Scope: wallet:transactions:read
      */
-    getTransactions(cb: (error: Error | null, result: Transaction[]) => void): void;
+    getTransactions(opts: {}, cb: (error: Error | null, result: Transaction[]) => void): void;
```

Comment:
Was this just wrong or has the underlying library changed?  If the latter, should the version number in the header be updated?